src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -225,6 +226,15 @@ class AuthScreens extends React.Component {
                 modal subscreens e.g. `/settings/profile` and this will allow us to navigate while inside the modal. We
                 are also using a custom navigator on web so even if a modal does not have any subscreens it still must
                 use a navigator */}
+                <RootStack.Screen
+                    name="WorkspacePeople"
+                    options={{
+                        headerShown: false,
+                        title: 'BETA - WORKSPACE PEOPLE',
+                    }}
+                    component={WorkspacePeopleModalStackNavigator}
+                    listeners={modalScreenListeners}
+                />
```

Comment:
This is probably annoying (since I asked you to basically do the opposite), but since we have merged Rajat's PR and the changes are ready now I think we can just add this screen here instead of create the `WorkspacePeopleModalStackNavigator`.

https://github.com/Expensify/Expensify.cash/blob/b55729a4c27deb3ea97566aadcb2b6724a147387/src/libs/Navigation/AppNavigator/WorkspaceSettingsDrawerNavigator.js#L11-L17

Sorry about the confusion there. Mostly I was trying to communicate that it would be a good practice to avoid including unmerged changes in a PR (since it could potentially lead to a lot of changes for you and make reviewers have to look at things that are unrelated), but now that those changes **are merged** it does make sense to use them since we're introducing changes that we will need to immediately undo.

Reply:
Of course, that is no problem, it was my mistake to include Rajat commits in here as well. I will merge `main` branch into this branch and update it so the page is part of the settings modal. Gonna do that later.